docs/spec/src/integration/spec/5-lifecycle-phases.md

```diff
@@ -90,77 +90,95 @@ message BlobStatusReply {
 
-The proxy can be configured to retry `FAILED` dispersal n times, after which it returns to the rollup a `503` HTTP status code which rollup batchers can use to failover to EthDA. See [here](https://github.com/ethereum-optimism/specs/issues/434) for more info.
+Once `BlobStatus.COMPLETE` is returned, the `signed_batch` and `blob_verification_info` fields are guaranteed to be included and should be used to construct the final `DACert`.
+
+ Any other terminal status indicates failure, and a new blob dispersal will need to be made.
+
+**Failover to Native Rollup DA**
+
+The proxy can be configured to retry `FAILED` dispersal `n` times, after which it returns to the rollup a `503` HTTP status code which rollup batchers can use to failover to EthDA or native rollup DA offerings (e.g, arbitrum anytrust). See [here](https://github.com/ethereum-optimism/specs/issues/434) for more info on the OP implementation and [here](https://hackmd.io/@epociask/SJUyIZlZkx) for Arbitrum. 
 
 ## BlobStatusReply → Cert
 
-This is not necessarily part of the spec but is currently needed given that the disperser doesn’t actually return a cert, so we need a bit of data processing to transform its returned value into a Cert. The transformation is visualized in the [Ultra High Res Diagram](../spec.md#ultra-high-resolution-diagram). The main difference is just calling the [`getNonSignerStakesAndSignature`](https://github.com/Layr-Labs/eigenda/blob/d9cf91e22b6812f85151f4d83aecc96bae967316/contracts/src/core/EigenDABlobVerifier.sol#L222) helper function within the new `EigenDACertVerifier` contract to create the `NonSignerStakesAndSignature` struct. The following pseudocode below exemplifies this necessary preprocessing step:
+This is not necessarily part of the spec but is currently needed given that the disperser doesn't actually return a cert, so we need a bit of data processing to transform its returned value into a Cert. The transformation is visualized in the [Ultra High Res Diagram](../spec.md#ultra-high-resolution-diagram). 
 
-```python
+In the updated implementation, a `CertBuilder` constructs the DA Cert through direct communication with the `OperatorStateRetriever` contract, which provides the necessary information about operator stake states. This approach ensures accurate on-chain data for certificate verification. The following pseudocode demonstrates this process:
 
+```python
 class CertV2:
-    batch_header: any  # You can specify proper types here
+    batch_header: any
     blob_verification_proof: any
     nonsigner_stake_sigs: any
+    cert_version: uint8  # Updated to use uint8 instead of uint256
```

Comment:
think we only have the uint8 cert_version on v3 right? 
Also this CertV2 is confusing right given that we have CertV3 already and this document actually describes CertV3. Should we just say Cert?

Reply:
we should probably instead say `DACert` given its the glossary term used through the md book

Reply:
[3a4077a](https://github.com/Layr-Labs/eigenda/pull/1642/commits/3a4077afb0ba4489162877d085473b947df611e6)